mk_verificator/utils/logger.py

```diff
@@ -0,0 +1,61 @@
+import os
+import logging
+import paramiko
+
+from logging import StreamHandler
+from logging.handlers import RotatingFileHandler
+
+
+_log_dir = "/var/log/mk_verificator"
```

Comment:
good approach, then let's first of all create this global config. I suggest to discuss it tomorrow after demo-call